src/genericobject.cpp

```diff
@@ -182,3 +182,15 @@ std::string gob_cmd_update_nametag_attributes(video::SColor color)
 	writeARGB8(os, color);
 	return os.str();
 }
+
+std::string gob_cmd_update_infant(u16 id, u8 type, std::string client_initialization_data)
```

Comment:
I think this should remain as it is, so that it remains consistent with all the other functions that genericobject.cpp provides.